src/app/components/isAnswerCorrect.js

```diff
@@ -0,0 +1,11 @@
+const isAnswerCorrect = (guess,correct) =>{
```

Comment:
Consider changing name of second param, as `correct` (for me) is more like result of comparison, rather than correct answer
```suggestion
const isAnswerCorrect = (guess, answer) =>{
```

Reply:
Good point, thanks!